lib/workers/branch/index.ts

```diff
@@ -462,6 +488,38 @@ export async function processBranch(
     await setStability(config);
     await setConfidence(config);
 
+    if (branchExists && stopRebasingLabelPresents && prRebaseChecked) {
+      // Uncheck the Rebase checkbox
+      let newBody = branchPr.body?.replace(
+        '- [x] <!-- rebase-check -->',
+        '- [ ] <!-- rebase-check -->'
+      );
+
+      if (newBody.indexOf('stopRebasingLabel') === -1) {
+        // Append the description line
+        newBody = newBody.replace(
+          'click this box',
+          emojify(
+            'check this box.\n\n :memo: **Note**: The rebase/retry checkbox was unchecked because of `stopRebasingLabel` setting in your configuration'
+          )
+        );
+      }
+
+      logger.debug(
+        'Updating existing PR to indicate that further rebasing is turned off'
+      );
+
+      await platform.updatePr({
+        number: branchPr.number,
+        prTitle: branchPr.title,
+        prBody: newBody,
+        platformOptions: getPlatformPrOptions(config),
+      });
+
+      // This flag may be used by ensurePr to generate the correct PR body
+      config.prRebaseBoxUnchecked = true;
+    }
+
```

Comment:
This is too hacky of a way to do it. It should not need its own `platform.updatePr()` and instead should leverage the existing logic